layouts/default.vue

```diff
@@ -41,45 +53,17 @@ export default {
         window.location.reload()
       }
     })
+  },
+  methods: {
+    parseRoute() {
+      const { name } = this.$route
+      const finalName = name.split('__')[0]
+      this.currentRoute = finalName
+    }
   }
 }
 </script>
 
 <style lang="postcss">
-html {
-  font-family: 'Source Sans Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI',
-    Roboto, 'Helvetica Neue', Arial, sans-serif;
-  font-size: 16px;
-  word-spacing: 1px;
-  -ms-text-size-adjust: 100%;
-  -webkit-text-size-adjust: 100%;
-  -moz-osx-font-smoothing: grayscale;
-  -webkit-font-smoothing: antialiased;
-  box-sizing: border-box;
-}
-
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-  margin: 0;
-  outline: none;
-}
-
-.nav-left {
-  left: -100%;
-  transition: 0.3s ease-out;
-}
-
-.nav-left.open {
-  display: inherit;
-  left: 0;
-  transition: 0.3s ease-out;
-}
-
-@media (min-width: 1024px) {
-  .nav-left {
-    position: inherit;
-  }
-}
+@import '../assets/css/global.css';
```

Comment:
could we not use `@/assets/css/global.css` ??

Reply:
Aliases in CSS are with `~` instead of `@`

Reply:
ye i will update

Reply:
i like to use the global css one. As we need this part in storybook too. I like to DRY ^^